.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+### Description of Changes
+What has been changed?
+
+### Documentation
+Provide links to all relevant documentation that should be updated before asking for the PR to be reviewed. If this section is not relevant please write 'Not Applicable' rather than deleting the section.
+
+### Risks & Impacts
+The potential risks of this change as a bullet pointed list and how it might impact the wider application. Please consider how easily it would be to rollback these changes if they cause problems.
```

Comment:
```suggestion
The potential risks of this change as a bullet pointed list and how it might impact the wider sdk. Please consider how easily it would be to revert these changes if they cause problems. Are there breaking changes?
```